packages/lambda-powertools-eventbridge-client/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@dazn/lambda-powertools-eventbridge-client",
+  "version": "1.15.6",
```

Comment:
Why start at 1.15.6 for a new package?

Reply:
dunno, lerna did that, I think it uses the version in the lerna config, probably doesn't hurt though as it keeps inline with the other packages